sprintMission4_/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "sprintmission4_",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "dev": "nodemon src/app.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "type": "module",
+  "dependencies": {
+    "@prisma/client": "^6.15.0",
+    "bcrpyt": "^2.0.0",
+    "bcrypt": "^6.0.0",
+    "cookie-parser": "^1.4.7",
+    "cookieparser": "^0.1.0",
```

의존성 관리는 중요해요.
존재하지 않는 패키지가 설치자체가 되지 않는 문제가 발생하거나,
미사용하는 패키지가 같이 혼재되어있으면 번들링 크기가 증가되고, IDE에서 코드 작성 시 혼동이 발생할 수 있어요.

추가로, 사용하지 않는 패키지에서 보안 취약점의 잠재적인 위험 요소도 있습니다.

```suggestion
    "bcrypt": "^6.0.0",
    "cookie-parser": "^1.4.7",
```